### ISSUE TITLE
Add feature to open the receipt link in browser.

### ISSUE THREAD

**Issue by EGOR-IND** [openMF/mobile-wallet#1220]
**Summary:**

The receipt link given in receipt activity is not doing anything by itself, it even has the color of a link which makes it look like a working link to a user, but it's not clickable.

**Steps to reproduce:**

1. Go to receipt activity
2. click on receipt link

**Expected behavior:**

It should open that link in a browser.

**Observed behavior:**

Right now it's not clickable.

**Device and Android version:**

VIVOY83, Android Version = 8.1.0

**Screenshots:**

<img src="https://user-images.githubusercontent.com/56648862/107517393-b8585400-6bd3-11eb-933a-374400df1cd4.jpeg" width="200" height="400"/>

**Comment by EGOR-IND** [comment]
I'm working on this issue.

**Comment by EGOR-IND** [comment]
@haran2248 @devansh-299 can you give your views on this issue.

**Comment by haran2248** [comment]
Actually the given link is just a deeplink even if you make it open in the browser it is of no use i guess.Only if the person has the app the deeplink will open.But in this case the share and copy feature has been implemented so i think it is of no use.
@divyank00 @devansh-299 Please confirm

**Comment by EGOR-IND** [comment]
Yes it is a deeplink though, but the link is actually there and even if we share or copy it what use will it be, I was actually thinking that if we're copying and sharing the link then there should be a way of accessing that link through app.

**Comment by haran2248** [comment]
Yeah but if you put it in a browser.It will show invalid address.If the user has the app it will open in the app.Opening it through chrome Im not sure if it is a good idea,But you can try making a PR anyway,It might be useful :)

**Comment by EGOR-IND** [comment]
No I'm saying we should open it in the app, there is a way I think to open a deeplink in the app. Let me do some research on it and then we'll talk😊.

**Comment by therajanmaurya** [comment]
Fixed
